order/views.py

```diff
@@ -84,4 +86,62 @@ def delete(self, request):
             return JsonResponse({'MESSAGE' : "Cart does not exist"}, status=400)
 
         except KeyError:
-            return JsonResponse({"MESSAGE" : "KEY_ERROR"}, status=400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE" : "KEY_ERROR"}, status=400)
+
+
+class PaymentView(View):
+    @check_user
+    def patch(self, request):
+        try:
+            data          = json.loads(request.body)
+            user          = request.user
+            user_address  = user.address
+            order_address = data.get('address')
+
+            if user_address is None:
+                User.objects.filter(id=user.id).update(address=order_address)
+                return JsonResponse({"message": "SUCCESS"}, status=200)
+
+            else:
+                order_address = user_address
+                User.object.filter(id=user.id).update(address=order_address)
+                return JsonResponse({"message": "SUCCESS"}, status=200)
+
+        except KeyError:
+            return JsonResponse({"error": "INVALID_ADDRESS"}, status=400)
+
+
+    @check_user
+    def get(self, request):
+        try:
+            user    = request.user
+            carts   = Cart.objects.filter(user=user)
+            coupons = UserCoupon.objects.filter(user=user).select_related('coupon')
```

Comment:
정확히는 `Coupon`객체가 아닌 User.Coupon 레코드를 조회하므로, 아래와 같이 수정해주시는게 명확할 것 같습니다.
```suggestion
            user_coupons = UserCoupon.objects.filter(user=user).select_related('coupon')
```

또한, User와 Coupon이 ManyToMany 관계라면, `couponse = user.coupon.all()`로 모든 쿠폰데이터를 가져올 수 있습니다.
이에 맞게 line 131~134를 수정해보시면 가독성이 좋아질 것 같습니다.

Reply:
확인했습니다 감사합니다~